client/modules/ftml-wasm-worker/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ftml-wasm-worker",
+  "license": "agpl-3.0-or-later",
+  "description": "A web workerized version of ftml-wasm",
+  "version": "0.0.0",
+  "keywords": [
+    "wikijump"
+  ],
+  "scripts": {
+    "clean": "trash dist .ultra.cache.json",
+    "build": "node ../../scripts/build-modules.js -self"
+  },
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/scpwiki/wikijump.git"
+  },
+  "bugs": {
+    "url": "https://scuttle.atlassian.net/servicedesk/customer/portal/2"
+  },
+  "homepage": "https://github.com/scpwiki/wikijump#readme",
```

Comment:
Not relevant to this PR, but this reminds me that at some point we're going to need to change all these to https://www.wikijump.org/

Reply:
Yeah a lot of the "meta" client repo stuff will need to be looked at. The repo is in a very nebulous state right now